docs/getting-started/deploy-operator.md

```diff
@@ -8,36 +8,6 @@ This documentation covers how to set up the communication between your client
 application and the Responsive control plane, as well as how to deploy the
 operator in your kubernetes context.
 
-## Prerequisites
```

Comment:
Gotcha, I'll add this to the Prereq section instead of removing it entirely

Reply:
@agavra are the api key/secret used here going to be the same as the one used below (in the `Configure Secrets` section)

in other words, these two configs will have the same value:
```
responsive.platform.api.{key/secret} == responsive.metrics.{api.key/secret}
```

If so, as an aside: it might be good to align the naming scheme when we do our config cleanup so that they both have `api` in both the key and secret config name (vs just `responsive.metrics.secret`)

Reply:
cool, thanks -- though I stand by my statement that they should use the same naming scheme (`x.y.z.api.{key/secret}` vs `x.y.z.{.api.key/secret}`

Reply:
yup I agree!